### ISSUE TITLE
Support Windows

### ISSUE THREAD

**Issue by umanwizard** [polarsignals/rust-jemalloc-pprof#14]
Cc @roblabla

**Comment by roblabla** [comment]
Yeah I'm working on this. It turned out to require changes across the stack:

- In jemalloc itself:
  - I had to add support for MSVC backtracking so the profiling feature worked: https://github.com/JustRustThings/jemalloc/commit/9e2edd7861a04d87ce8383af250b9099ba545869
  - I had to change how weak symbols are handled for malloc_conf so it supports MSVC toolchains: https://github.com/JustRustThings/jemalloc/commit/783dc0163face1420d6e03fa605e265fd6d56600
- In jemallocator, I had to fix a lot of errors related to windows support in the build system: https://github.com/tikv/jemallocator/pull/99
- And in this repo, I need to add support for getting windows mappings.

**Comment by umanwizard** [comment]
That will be really awesome if you can get it working! Let me know if you need any help on this repo's side.
